network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html

```diff
@@ -1,8 +1,6 @@
-{% extends "./primary_page.html" %}
+{% extends "../primary_page.html" %}
 {% load wagtailcore_tags wagtailimages_tags %}
 
-{% block bodyID %}initiatives{% endblock%}
-
```

Comment:
Similar to above; This is for CSS that might exist now to be applied to initiatives, but as we move forward we should only target CSS by page template, not page title.